packages/toolkit/connector-kit/src/types/passwordless.ts

```diff
@@ -4,27 +4,42 @@ import { z } from 'zod';
 
 import { type BaseConnector, type ConnectorType } from './foundation.js';
 
-export enum VerificationCodeType {
+export enum TemplateType {
```

Comment:
`VerificationCodeType` is used in other connectors including non-official connectors, will this be a breaking change?

Reply:
right. we can keep the original type though